src/WebAPIContentService/Infra/Data/Repository/MaterialRepository.cs

```diff
@@ -0,0 +1,57 @@
+﻿using WebAPIContentService.Domain.Entities;
+using WebAPIContentService.Infra.Data.Context;
+using WebAPIContentService.Infra.Data.Repository.Interfaces;
+using Microsoft.EntityFrameworkCore;
+using System.Xml.Linq;
+
+namespace WebAPIContentService.Infra.Data.Repository
+{
+    public class MaterialRepository : IMaterialRepository
+    {
+        private readonly ContentContext _context;
+
+        public MaterialRepository(ContentContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<Material> GetMaterialByIdAsync(int id)
```

Comment:
```suggestion
        public async Task<Material?> GetMaterialByIdAsync(int id)
```

Acredito que faça sentido colocar a ? aqui, já que o retorno pode ser nulo caso não exista, isso se reflete na interface, mas não tenho certeza

Reply:
Não sabia como solucionar isso, valeu!